README_v1.md

```diff
@@ -0,0 +1,129 @@
+# Stackable Operator for Apache Kafka
+
+[![Build Actions Status](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly/badge/icon?subject=Integration%20Tests)](https://ci.stackable.tech/job/kafka%2doperator%2dit%2dnightly)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://GitHub.com/stackabletech/kafka-operator/graphs/commit-activity)
+
+This is a Kubernetes Operator to manage [Apache Kafka](https://kafka.apache.org/) clusters.
+
+Looking to dive in? It's easy to get started with the Stackable data platform using `stackablectl`:
+
+```
+stackablectl spin up a demo
+```
+
+Read more in [link to fancy demo using stackablectl](#).
+
+## Installation
+
+To get the best experience, we recommended way to get started with Stackable is through the `stackablectl` tool.
+
+Read [the instructions below](#) to get started.
+
+## Getting Started
+
+To create a Kafka cluster with three nodes, you can follow this [tutorial](https://docs.stackable.tech/kafka/stable/getting_started/first_steps.html).
+
+Otherwise, give it a try with the [stackablectl](https://docs.stackable.tech/home/stable/getting_started.html) CLI tool!
+
+## Documentation
+
+The documentation for this operator can be found at <https://docs.stackable.tech/kafka/stable/index.html>.
+
+The documentation for all Stackable products can be found at <https://docs.stackable.tech>.
+
+## What Does This Do?
+
+Operator looks at custom resources in the cluster, and makes sure apps are started according to those custom resources.
```

Comment:
This is a different angle than I took, a bit more general (There's a resource and the operator deploys stuff). I think both has its place, I'm not so sure how much previous knowledge we should assume. Maybe we could have something more surface-level here and link to detailed explanations in the docs?

Reply:
Yeah, maybe it's too high-level. We could assume that somebody stumbling on this knows what an operator is.